### ISSUE TITLE
Remove reliance on aes_q() in ggplot2 calls and implement tidy evaluation

### ISSUE THREAD

**Issue by Rekyt** [FRBCesab/funbiogeo#81]
`aes_()` and `aes_q()` has been deprecated in ggplot2 3.0.0 as stated in the [news blogpost](https://www.tidyverse.org/blog/2018/07/ggplot2-3-0-0/#tidy-evaluation), we should remove our reliance on it and instead consider tidy evaluation.
This forces us to depend on ggplot2 3.0.0, but it has been release 4.5 years ago, so should be pretty safe.

**Comment by ahasverus** [comment]
Agree. With the latest version of `ggplot2` (3.4.2), this now throws an error.
